### DISCUSSION TITLE
My Example with cib works with GCC 9.3 but not with any clang version. Can someone help?

### DISCUSSION THREAD

**Discussion by tunguskar** [intel/compile-time-init-build#525] [Q&A] [answered]
Here is my example.

https://godbolt.org/z/cK46MYdxv

And here the same example with clang, which does not compile:

https://godbolt.org/z/bYv9ro35a

Maybe someone can help me on this topic?

**Comment by tunguskar** [comment]
I tried it locally with the last version of cib::cpp17 and there it works so maybe has something todo with an outdated version

**Comment by tunguskar** [comment]
Not needed anymore

**Reply by lukevalenty** [reply]
what was the resolution?

**Reply by tunguskar** [reply, marked as answer]
> what was the resolution?

The single Header Version, used in the godbolt exampels, is outdated. I did the Same in the cpp17 branch in a Linux wsl environment and IT worked. So no further Investigation on this
